commands/game/gwhost.js

```diff
@@ -0,0 +1,34 @@
+const db = require("quick.db")
```

Comment:
two commands with the same name don't work. It will always execute the file that was first added to the client.commands collection.
Delete this one and if there were any major changes try to implement them in the other file

Reply:
I actually accidentally added this. It's in the wrong folder as well but i've deleted them